.github/workflows/publish.yml

```diff
@@ -1,34 +1,35 @@
-name: npm-publish
-on:
-  push:
-    branches:
-      - master # Change this to your default branch
-jobs:
-  npm-publish:

```

Comment:
Can you avoid changing all the end lines ? It makes things hard to review

Reply:
i am sorry about the end lines, which ones do we want to use?, will stick to those from now on

Reply:
no more from now on